modules/tinymce/src/core/main/ts/api/AddOnManager.ts

```diff
@@ -12,71 +12,91 @@ import Editor from './Editor';
 import I18n from './util/I18n';
 
 /**
- * This class handles the loading of themes/plugins or other add-ons and their language packs.
+ * TinyMCE theme pseudo class. Allows for a custom theme to be used with TinyMCE when registered using the ThemeManager.
  *
- * @class tinymce.AddOnManager
- */
-
-/**
- * TinyMCE theme class.
+ * @summary This is a pseudo class that describes how to create a custom theme for TinyMCE.
+ * <br /><br />
```

Comment:
```suggestion
 * <br><br>
```
We're not using XHTML, so this should technically be a void element.